### DISCUSSION TITLE
Blueretro retroscaler ps2 adapter

### DISCUSSION THREAD

**Discussion by djmtech** [darthcloud/BlueRetro#964] [Q&A] [answered]
Hi I bought a retroscaler PS1-2 adapter and I went to update it via usb and follow the steps. I held the pair button down then connected the adapter and then used the nodeMCU flasher with the HW1 PS bin and it said it updated but when I unplugged it and then replugged the adapter in the sync light doesn't flash now its not work. I tried to reset it and everything both lights are solid. does anyone know how to fix this? Thank you

Instructions I followed:
https://www.retroscaler.com/wp-content/uploads/2023/06/Blueretro-PS1PS2-manual.pdf
https://github.com/RetroScaler/Bluereteo-PS?tab=readme-ov-file

bought from:
https://a.aliexpress.com/_mNyWtCS

**Comment by darthcloud** [comment, marked as answer]
Use the real instructions:
https://github.com/darthcloud/BlueRetro/wiki/Flashing-firmware-Windows-10

**Reply by djmtech** [reply]
Thank you I owe you a beer that fixed my issue. I really appreciate the quick response and help!!!!
